spec/swagger_schemas/models/participant_declaration.rb

```diff
@@ -141,6 +141,36 @@
     format: :"date-time",
     example: "2021-05-31T02:22:32.000Z",
   }
+  h[:v3][:properties][:attributes][:properties][:delivery_partner_id] = {
+    description: "The delivery partner ID",
+    type: :string,
+    format: :uuid,
+    required: true,
+    nullable: true,
+    example: "524df095-f9bf-4f9d-ba4c-772545a99e60",
+  }
+  h[:v3][:properties][:attributes][:properties][:secondary_delivery_partner_id] = {
```

Comment:
is it worth ordering these new fields so they match the `DeclarationSerializer` so they are in the same order the API returns?
so:
delivery_partner_id
delivery_partner_name
secondary_delivery_partner_id
secondary_delivery_partner_name